mri/operators/fourier/utils.py

```diff
@@ -255,37 +255,53 @@ def get_stacks_fourier(kspace_loc):
 
     Parameters
     ----------
-    ksapce_plane_loc: np.ndarray
+    kspace_plane_loc: np.ndarray
         the mask samples in the 3D Fourier domain.
-
+    shape: tuple
+        Reconstructed volume shape
     Returns
     ----------
-    ksapce_plane_loc: np.ndarray
+    kspace_plane_loc: np.ndarray
         A 2D array of samples which when stacked gives the 3D samples
     z_sample_loc: np.ndarray
         A 1D array of z-sample locations
     sort_pos: np.ndarray
         The sorting positions for opertor and inverse for incoming data
+    idx_mask_z: np.ndarray
```

Comment:
What do you mean by masking entire planes?
Masking actually means not acquiring so the doc should mention that it's possible to not acquire entire plane, is that what you mean?

Reply:
Yes, from what I understand now we have the possibility to undersample in the stack direction as well right?

Reply:
Yes we do. It is just the term mask that do not make that much sense. We do not really mask the 3D k-space since each plane is non-Cartesian therefore it cannot be considered as masked Fourier space. 
Beacause of this specific point the following documentation is false
https://github.com/CEA-COSMIC/pysap-mri/blob/cf4dff5dcb28058d946fbf210e584ededa727ec8/mri/operators/fourier/utils.py#L258-L259
It should be replaced by 
```
Acquired 3D k-space locations such that the each plane is composed of the same non-Cartesian samples, 
while the third dimension correspond to a Cartesian under-sampling
```
Something like this

Reply:
I see !